src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/AdminController.cs

```diff
@@ -103,7 +103,6 @@ public async Task<IActionResult> Index(TenantIndexOptions options, PagerParamete
                        Description = x["Description"],
                        Name = x.Name,
                        ShellSettings = x,
-                       IsDefaultTenant = x.IsDefaultShell(),
```

Comment:
Why?

Reply:
Hmm yes because you now test it in a view

Reply:
Yup